runtimes/kotlin-1.9/src/main/kotlin/io/openruntimes/kotlin/Server.kt

```diff
@@ -28,6 +29,7 @@ suspend fun main() {
         .patch("/*") { runBlocking { execute(it) } }
         .options("/*") { runBlocking { execute(it) } }
         .head("/*") { runBlocking { execute(it) } }
+        .addHandler(HandlerType.TRACE, "/*") { runBlocking { execute(it) } }
```

Comment:
as per the docs, https://javalin.io/documentation#handlers you can use `trace()` directly.